scripts/vim-patch.sh

```diff
@@ -162,22 +167,17 @@ list_vim_patches() {
   # runtime patches before between 384 and 442 have already been ported
   # to Neovim as of the creation of this script.
   local vim_commits=$(cd "${VIM_SOURCE_DIR}" && \
-    hg log --removed --template='{if(startswith("Added tag", firstline(desc)),
-      "{latesttag}\n",
-      "{if(startswith(\"updated for version\", firstline(desc)),
-        \"\",
-        \"{node}\n\")}")}' -r tip:v7-4-442)
+    git log --pretty='tformat:%H|%d' v7.4.442..HEAD | awk -f ${NEOVIM_SOURCE_DIR}/scripts/vim-patch-helper.awk)
+
   # Append remaining vim patches.
   # Start from 7.4.160, where Neovim was forked.
   local vim_old_commits=$(cd "${VIM_SOURCE_DIR}" && \
-    hg log --removed --template='{if(startswith("Added tag",
-      firstline(desc)),
-      "{latesttag}\n")}' -r v7-4-442:v7-4-161)
+    git log --pretty='tformat:%d' v7.4.160..v7.4.442 | awk '{ if ($1 != "") { gsub(/[()]/, "", $2); print($2); } }')
```

Comment:
See https://gist.github.com/e26aba8e38f56d02a6f8 the output of this and the other similar git log line.

For this line in particular, IIRC the point was that with the runtime reintegration, the tests for whether non-patch runtime revs had been integrated didn't really work.  So for the older revs the idea was to extract only the tag/patch names and leave off pulling the non-patch commit hashes.